src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp

```diff
@@ -669,9 +669,11 @@ namespace NativeMsh
     static PCSTR trustedAssemblies[] =
     {
         "Microsoft.ApplicationInsights",
-        "Microsoft.CodeAnalysis",
         "Microsoft.CodeAnalysis.CSharp",
+        "Microsoft.CodeAnalysis",
+        "Microsoft.CodeAnalysis.VisualBasic",
```

Comment:
Do we want to include this one? I think we will remove VB support before 6.1 RC. If we include it here now, we will have to update the windows.psrp package again when removing the VB support.

Reply:
Will remove.  We can always add it back later if needed.